shacl12-core/index.html

```diff
@@ -4517,6 +4520,462 @@ <h4>sh:uniqueLang</h4>
+						<div class="shapes-graph">
+							<div class="turtle">
+ex:PersonShape
+	a sh:NodeShape ;
+	<span class="target-can-be-skipped">sh:targetClass ex:Person ;</span>
+	sh:property [
+		sh:path ex:hobbies ;
+		sh:maxListLength 2 ;
+	] .
+							</div>
+							<div class="jsonld">
+								<pre class="jsonld">{
+	"@id": "ex:PersonShape",
+	"@type": "sh:NodeShape",
+	"sh:property": {
+		"sh:maxListLength": {
+			"@type": "xsd:integer",
+			"@value": "2"
+		},
+		"sh:path": {
+			"@id": "ex:hobbies"
+		}
+	},
+	"sh:targetClass": {
+		"@id": "ex:Person"
+	}
+}</pre>
+							</div>
+						</div>
+						<div class="data-graph">
+							<div class="turtle">
+ex:person1 a ex:Person ;
+	ex:hobbies ( "reading" "writing" ) .
+
+<span class="focus-node-error">ex:person2</span> a ex:Person ;
+	ex:hobbies ( "reading" "writing" "swimming" ) .
+							</div>
+							<div class="jsonld">
+								<pre class="jsonld">{
+	"@graph": [
+		{
+			"@id": "ex:person1",
+			"@type": "ex:Person",
+			"ex:hobbies": {
+				"@list": [
+					"reading",
+					"writing"
+				]
+			}
+		},
+		{
+			"@id": "ex:person2",
+			"@type": "ex:Person",
+			"ex:hobbies": {
+				"@list": [
+					"reading",
+					"writing",
+					"swimming"
+				]
+			}
+		}
+	]
+}</pre>
+							</div>
+						</div>
+					</aside>
+				</section>
+				
+				<section id="UniqueMembersConstraintComponent">
+					<h4>sh:uniqueMembers</h4>
+					<p>
+						<code>sh:uniqueMembers</code> specifies whether <a href="#syntax-rule-SHACL-list">SHACL list</a> <a>value nodes</a> must have unique members.
+					</p>
+					<p>
+						<span class="component-class">Constraint Component IRI</span>: <code>sh:UniqueMembersConstraintComponent</code>
+					</p>
+
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<tr>
+							<th>Property</th>
+							<th>Summary and Syntax Rules</th>
+						</tr>
+						<tr>
+							<td><code>sh:uniqueMembers</code></td>
+							<td>
+								A boolean that specifies whether the members of the <a href="#syntax-rule-SHACL-list">SHACL list</a> must be unique.
+								<span data-syntax-rule="uniqueMembers-datatype">The values of <code>sh:uniqueMembers</code> in a shape are literals with datatype <code>xsd:boolean</code>.</span>
+							</td>
+						</tr>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="UniqueMembers">
+							Let <code>$uniqueMembers</code> be a <a>parameter value</a> for <code>sh:uniqueMembers</code>.
+							Each <a>value node</a> <code>v</code> must be a <a href="#syntax-rule-SHACL-list">SHACL list</a> - if <code>v</code> is not a SHACL list there is a <a>validation result</a>.
+							If <code>$uniqueMembers</code> is <code>true</code> and the list <code>v</code> has duplicate members,
+							there is a <a>validation result</a>. There must be at most one <a>validation result</a> for each list <code>v</code>.
```

Comment:
The sentence "There must be at most one validation result..." should be removed because it is redundant and may not apply in cases where the same value node is reached by different, violated constraints.

Reply:
Resolved in https://github.com/w3c/data-shapes/commit/ca91c2c71ce6526995d39a5ba2eca744ba2d9d39